### ISSUE TITLE
Jumping into development a little late in the game

### ISSUE THREAD

**Issue by byumark** [kentcdodds/ama#293]
I graduated from college about 15 years ago with a degree not related to development. Over the first 8 years of my career I realized that I am more of a technical person and moved to a job where I was to be a systems engineer. However, need and interest had me getting involved with development work in that position. Scripting, System Integrations, Small Internal Web Apps, etc. I have loved doing development.

I'm still on a team of mostly Systems Engineers but I do mostly development. My interest is now very much in development, but I've been mostly self-taught and never formally trained. Also, because I'm not on a team of developers, I feel like my access to resources and help is limited as well as the opportunity to grow by learning from others around me.

So I've considered going back to school for a masters in computer science/software development, attending bootcamps, looking for opportunities to work with others, and/or continuing to learn on my own.  The end goal is for sure to better myself as a developer. Whether that is for my current position or to eventually get a developer job with a team of developers, I don't really know.

What would you suggest I do and/or not do?

Thanks for your time.

**Comment by morajabi** [comment]
Good question. Hope @kentcdodds answer this.

**Comment by kentcdodds** [comment]
Hey @byumark :wave:

Thanks for the question. One of the things I love about the tech industry is how fast it moves. It moves so fast that the knowledge of tools and techniques you learn today will be antiquated in only a few years. This means that people who have been in tech for 20 years don't have as HUGE an advantage over you as you think.

From my own experience, I was too arrogant to consider that I couldn't teach anyone anything and started speaking internationally within a year of graduating from college. There's no shortcut to experience, so I did spend a LOT of time coding, building, and teaching things. But if you do that then you'll find success.

> So I've considered going back to school for a masters in computer science/software development, attending bootcamps, looking for opportunities to work with others, and/or continuing to learn on my own. The end goal is for sure to better myself as a developer. Whether that is for my current position or to eventually get a developer job with a team of developers, I don't really know.

I just answered a question I got from someone in an email that addresses this actually. See #306 

TL;DR, I probably wouldn't suggest you go to school to get a masters degree...

You might also be interested in this answer: https://github.com/kentcdodds/ama/issues/89

Good luck!

**Comment by byumark** [comment]
Thanks for the input! I will check out the links you included.
